gdx/src/com/badlogic/gdx/input/RemoteInput.java

```diff
@@ -457,6 +457,10 @@ public void getTextInput (TextInputListener listener, String title, String text,
 	public void setOnscreenKeyboardVisible (boolean visible) {
 	}

```

Comment:
Suggestion: It could call `setOnscreenKeyboardVisible(visible, Default)`

Reply:
Is there any reason for it to here? It looks like RemoteInput wouldn't have any relation to showing a keyboard.

Reply:
@PokeMMO no specific reason, if in the future someone wants to implement, should only care about one method. It's only a suggestion.

Reply:
I'd rather leave it as is to match all of the other input types which do not support it.